docs/install-standalone.md

```diff
@@ -0,0 +1,57 @@
+<!--
+title: Installing Serverless Framework as a standalone binary
+layout: Doc
+-->
+
+# Installing Serverless Framework as a standalone binary
```

Comment:
This new page is intentionally not in the menu.

It can be accessed via the "Setup" page.